Update/_tsum_003_2.txt

```diff
@@ -943,7 +943,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 3, "ps3/s09/03/170300004", 128, TRUE);
 	OutputLine(NULL, "「……圭ちゃんってＫって呼ばれてるの？",
-		   NULL, "\"...Kei-chan, he's calling you 'K'.", Line_WaitForInput);
+		   NULL, "\"...Kei-chan, he's calling you 'K.'", Line_WaitForInput);
```

Comment:
Both of these are valid, depending on where you are.  Personally, I prefer the original British/Wikipedia-style `'K'.`

Reply:
It was just for consistency, but your choice

Reply:
If I remember correctly you changed it everywhere in the question arcs because you thought it was invalid. So now I need to write a script to revert it.

Reply:
Oh no, it was that sometimes they were inside, sometimes not, so I just wanted to keep it consistent.
It's fine either way, but just having it the same everywhere was the goal